test/cases/inherit_want_safety.zig

```diff
@@ -35,3 +35,4 @@ pub export fn entry() usize {
 
 // compile
 // output_mode=Obj
+// emit_bin=false
```

Comment:
FYI @andrewrk 5beb5f20d282fd0db8715899ba6a5d6630d91bb4 you'll probably want to revert this one once you finish your current wasm linker work.